### ISSUE TITLE
Cannot build using build_runner

### ISSUE THREAD

**Issue by Ben-Baert** [Tienisto/flutter-fast-i18n#13]
When I try to generate the strings.g.dart file using the build_runner, I get the following error:

```
[SEVERE] fast_i18n:i18nBuilder on lib/presentation/core/i18n/strings.i18n.json:

This builder requires Dart inputs without syntax errors.
However, package:brickfact/presentation/core/i18n/strings.i18n.json (or an existing part) contains the following errors.
strings.i18n.json:1:1: Expected a method, getter, setter or operator declaration.
```

When I try with the `flutter pub run fast_i18n` command it works without issues. It would however be nice if it works with the build_runner, even if it is slower, as this command is always running. Is there a way to make this work?

**Comment by Tienisto** [comment]
What Flutter and FastI18n version do you use?

**Comment by Tienisto** [comment]
Thanks for this issue. I could reproduce this. It should be fixed in version `1.7.1`

**Comment by Ben-Baert** [comment]
I just tried it; it's fixed indeed. Thanks a lot for the quick response and fix!
